docs/getting-started/first-app.md

```diff
@@ -66,33 +66,33 @@ All changes are saved automatically.
 
 ### Create the Description Field
 
-1. In the *Entity Fields* text box type “Description”.
```

Comment:
@GalCegla per the GitHub syntax guide, both a single asterisk and the underscore result in italics. Why are you changing all the italics?

Reply:
I know, but the editor did it automatically 😅

Reply:
@GalCegla which editor? 
The problem is that I need to check all the changes [that just this was changed], so it consumes time.

Reply:
VScode

Reply:
טוב, אז אין מה לעשות. 
זה מה יש.

Reply:
Is because Prettier enforces a deterministic standard on markdown. I suggest we use it for all documents to enforce consistency. There is also a nice [Chrome extension](https://chrome.google.com/webstore/detail/prettier/fbcfnmplppajblbmdehballiekfgdkhp) that adds a formatting button to GitHub.

Reply:
What does Prettier enforce @iddan and what will installing the Chrome extension do?